ksp_plugin/plugin.cpp

```diff
@@ -1183,13 +1183,18 @@ Plugin::NewBodySurfaceNavigationFrame(
 }
 
 not_null<std::unique_ptr<PlottingFrame>>
-Plugin::NewRotatingPulsatingPlottingFrame(Index const primary_index,
-                                          Index const secondary_index) const {
+Plugin::NewRotatingPulsatingPlottingFrame(
+    Index const primary_index,
+    std::vector<Index> const secondary_indices) const {
```

Comment:
Should this be a `flat_hash_set`?  If the indices repeat, something is very wrong.